airbyte-webapp/src/components/connection/CatalogTree/StreamHeader.module.scss

```diff
@@ -58,7 +58,7 @@
 }
 
 .purpleBackground {
-  background-color: colors.$blue-transparent;
+  background-color: colors.$blue-40;
```

Comment:
`blue-40` is basically the full-opacity equivalent of `blue-transparent`
![image](https://user-images.githubusercontent.com/7550957/201698650-c67debbf-dde4-4893-aa80-e3ed0fa23331.png)